docs/src/components/Sandbox.tsx

```diff
@@ -2,24 +2,26 @@ import { Flow } from '@flows-ai/ui'
 import s from 'dedent'
 import { useEffect, useState } from 'react'
 
-const defaultCode = s`
+const codeWrapping = (flow: string) => s`
   import { parallel } from 'https://esm.sh/flows-ai/flows';
 
-  export default parallel([
+  export default ${flow}
+`
+
+const initialFlow = s`parallel([
```

Comment:
What was wrong with the previous one? 🤔 Do you plan to utilize `codeWrapping` function somewhere else later?

Reply:
Now there's no import and export displayed in the code editing in textarea. Just the functions and flows.

Reply:
True, but that is somewhat magical, isn't it? I was thinking that we could allow named exports in the future (multiple) as that closely reflects the realities of a `flow` file, in this case, we could always display tabs in the UI, where you can browse those named exports!